titan/features/haart.py

```diff
@@ -152,11 +150,11 @@ def get_transmission_risk_multiplier(self, time: int, interaction_type: str):
             params = self.agent.location.params
             if interaction_type == "injection":
                 prob = params.partnership.injection.transmission.haart_scaling[
-                    self.adherence
+                    self.adherent
```

Comment:
does this still work now that adherent is true/false instead of ints?

Reply:
Yes, python treats the true/false as 1/0